src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -480,8 +480,16 @@ def consRecordAge(self, rse):
             raise MSUnmergedPlineExit(msg)
 
         isConsDone = self.rseConsStats[rseName]['status'] == 'done'
-        isConsNewer = self.rseConsStats[rseName]['end_time'] > self.rseTimestamps[rseName]['prevStartTime']
+        if not isConsDone:
+            msg = "RSE: %s In non-final state in Rucio Consistency Monitor. " % rseName
```

Comment:
How about a minor change to this message, like:
```
msg = "RSE: {} has a non-final Rucio ConMon status: {}. ".format(rseName, self.rseConsStats[rseName]['status'])
```
?

Reply:
good suggestion indeed. 
Adding this in a second.